_includes/neurips/2019/presenters-info.nips.html

```diff
@@ -0,0 +1,28 @@
+ <!-- Presenters Modals -->
```

Comment:
Please rename this file to follow the updated naming of the conference and other files for this year to NeurIPS from NIPS. Also, include the year to follow the naming convention of the rest of the include files for this year for easier searchability. 
The file name should be: presenters-info.neurips.2019.html